shared/featureconfig/flags.go

```diff
@@ -179,4 +184,5 @@ var BeaconChainFlags = append(deprecatedFlags, []cli.Flag{
 	enableSnappyDBCompressionFlag,
 	enablePruneBoundaryStateFlag,
 	fastCommitteeAssignmentsFlag,
+	connectToSlasherFlag,
```

Comment:
This name can be better, enableSlasherFlag?

Reply:
changed